mrchecker/Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+@Library('pdehnel-sharedlib@feature/devon4j') _
+import com.capgemini.productionline.configuration.*
+
+Jenkins jenkinsConfiguration = new Jenkins();
+GitLab gitlabConfiguration = new GitLab(this, params.GITLAB_USER_PRIVATE_TOKEN)
+
+pipeline{
+
+    agent any
+
+    //Jenkins Job Parameters
+    parameters { 
+        string(name: 'GITLAB_USER_PRIVATE_TOKEN', defaultValue: '', description: 'NEEDS TO BE SET!. Private Token of a Production Line Gitlab User that can be used to create repositories.') 
+        string(name: 'GITLAB_CREATE_GROUP_NAME', defaultValue: 'devon', description: 'Private Token of a Production Line Gitlab User that can be used to create repositories.') 
+        string(name: 'GITLAB_CREATE_PROJECT_NAME', defaultValue: 'MrChecker', description: 'Private Token of a Production Line Gitlab User that can be used to create repositories.') 
+        string(name: 'GITLAB_CREATE_BRANCH', defaultValue: 'master', description: 'Private Token of a Production Line Gitlab User that can be used to create repositories.') 
+        string(name: 'GITLAB_CREATE_PROJECT_DESCRIPTION', defaultValue: 'automation test framework', description: 'Private Token of a Production Line Gitlab User that can be used to create repositories.') 
+    }
+
```

Comment:
Please have a look at the devon4j-mts template. Create a new stage here and install the required plugins/customtools.

Reply:
As discussed with Marvin documentation and Jenkinsfile are created for the state when we have new release which will contain new Jenkins image with needed plugins and configs, all the steps for current state are included in the documentation (steps which plugins to install and how to configure e.g Allure Commandline).

Reply:
No. please have a look at the devon4j-mts template. We install plugins there with a jenkins command.